specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2019-01-21/examples/KustoDataConnectionsCheckNameAvailability.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-123456789098",
+    "api-version": "2019-01-21",
+    "clusterName": "kustoProd",
+    "databaseName": "KustoDatabase8",
+    "dataConnectionName": "DataConnections8",
```

Comment:
This should be the body of the request:

{
  "name": "{resourceNameToVerify}",
  "type": "{fully qualified resource type which includes provider namespace}"
}

Reply:
Thanks, fixed